frontend/lib/src/AppNode.ts

```diff
@@ -840,6 +880,24 @@ export class AppRoot {
     )
   }
 
+  public removeTransientNodes(): AppRoot {
+    const main = this.main.removeTransientNodes()
+    const sidebar = this.sidebar.removeTransientNodes()
+    const event = this.event.removeTransientNodes()
+    const bottom = this.bottom.removeTransientNodes()
+
+    return new AppRoot(
+      this.mainScriptHash,
+      new BlockNode(
+        this.mainScriptHash,
+        [main, sidebar, event, bottom],
+        new BlockProto({ allowEmpty: true }),
+        this.main.scriptRunId
+      ),
+      this.appLogo
```

Comment:
Not sure, but maybe we also need the same logic for app logo as done in `clearSraleNodes` ?